webhook/tests/test_main.py

```diff
@@ -5,7 +5,76 @@
 
 client = TestClient(app)
 
+
 def test_index():
-    response = client.get('/')
-    assert response.status_code == 200
-    assert response.json() == {"data":"Hello World"}
+    r = client.get('/')
+    assert r.status_code == 200
+    assert r.json() == {"data": "Hello World"}
+
+
+correct_headers = {"X-Appengine-Inbound-Appid": "virustotal-step-2020"}
+
+
+def test_query_results_str():
+    r = client.post(
+        '/query-results/',
+        headers=correct_headers,
+        json={"data": "str"})
```

Comment:
Update tests to match correct data format

Reply:
Done